src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+
+  </head>
+  <body class="page">
+    <div class="card" data-qa="card">
+      <div class="card__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419"
+          class="card__image-img">
+      </div>
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__product-code">
+        Product code: 195434
+      </div>
+      <div class="card__rewievs">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__rewievs-title">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__price ">
+        <div class="price-title">
```

Comment:
```suggestion
      <div class="card__price ">
        <div class="card__price-title">
```
It's an element of card. Fix all cases